pkg/pillar/cmd/zedagent/zedagent.go

```diff
@@ -1693,6 +1698,7 @@ func handleAppInstanceStatusModify(ctxArg interface{}, key string,
 	uuidStr := status.Key()
 	PublishAppInfoToZedCloud(ctx, uuidStr, &status, ctx.assignableAdapters,
 		ctx.iteration)
+	triggerLocalAppInfoPOST(ctx.getconfigCtx)
```

Comment:
Shouldn't there also be a trigger call in the Delete function below so that the local server can see when an app instance has been deleted?

Reply:
Good point, added.